.gitignore

```diff
@@ -1,17 +1,20 @@
-*~
-*.pyc
-*.pyo
-*.swp
-parser.out
-parsetab.py
-.idea
+/.cache
 /.tox
+/.tox-*
+/.venv
+/.venv-*
 /build
 /dist
 /MANIFEST
-/stone.egg-info
 /setuptools-3.1-py2.7.egg
 /setuptools-3.1.zip
-/.cache
-.DS_Store
+/stone.egg-info
 .arcconfig
+.DS_Store
+.idea
+parser.out
+parsetab.py
+*.pyc
+*.pyo
+*.swp
+*~
```

Comment:
I can revert or change it if desired. I don't think the list is long enough to consider pattern matching performance within `git` (if list order is even important, which I don't know).